DEPRECATED.md

```diff
@@ -16,6 +16,7 @@ A logged warning is expected for each deprecated item that is in deprecation win
 * Use of `runtime_key` in `RequestMirrorPolicy`, found in
   [route.proto](https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/route/route.proto)
   is deprecated. Set the `runtime_fraction` field instead.
+* Use of buffer filter `max_request_time` is deprecated. in favor of http connection manager of :ref:request timeouts <envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.request_timeout>
```

Comment:
Sorry, but I don't think the rst-style links work in this file -- needs to be markdown: `[linktext](url)`